.github/workflows/pipelines-root.yml

```diff
@@ -316,6 +340,16 @@ jobs:
           infra_live_directory: "."
           deploy_branch_name: ${{ steps.gruntwork_context.outputs.deploy_branch_name }}
 
+      - name: Post Execute Custom Action
+        uses: ./pipelines-actions/.github/custom-actions/post-execute
+        with:
+          PIPELINES_GRUNTWORK_READ_TOKEN: ${{ steps.pipelines-gruntwork-read-token.outputs.PIPELINES_TOKEN }}
```

Comment:
similar to orchestrate - we should pass through the outputs of execute here.  `plan_folder` in particular is likely useful for customization purposes